ui/Modal/ManageRemotes.svelte

```diff
@@ -0,0 +1,145 @@
+<script lang="ts">
+  import { Button, Input } from "../DesignSystem/Primitive";
+  import { Illustration, List, Remote } from "../DesignSystem/Component";
+
+  import Peer from "./ManageRemotes/Peer.svelte";
+  import PeerFollowRequest from "./ManageRemotes/PeerFollowRequest.svelte";
+
+  import type { PeerId } from "../src/identity";
+  import type { Urn } from "../src/urn";
+
+  import { Variant as IllustrationVariant } from "../src/illustration";
+  import {
+    addPeer,
+    pendingPeers,
+    peerSelection,
+    peerValidation,
+    project as store,
+    removePeer,
+  } from "../src/project";
+
+  let newPeer: PeerId;
+
+  $: if (newPeer === "") {
+    peerValidation.reset();
+  }
+
+  const submitPeer = async (projectUrn: Urn) => {
+    if (await addPeer(projectUrn, newPeer)) {
+      newPeer = "";
+    }
+  };
+
+  const cancelFollowRequest = (projectUrn: Urn, peerId: PeerId) => {
+    removePeer(projectUrn, peerId);
+    peerValidation.reset();
+  };
+
+  const unfollowPeer = (projectUrn: Urn, peerId: PeerId) => {
+    removePeer(projectUrn, peerId);
+    peerValidation.reset();
+  };
+</script>
+
+<style>
+  .container {
+    width: 38.5rem;
+    background: var(--color-background);
+    border-radius: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem;
+  }
+
+  .container:focus {
+    outline: none;
+  }
+
+  .peer-entry-form {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+  }
+
+  .peer-entry-field {
+    display: flex;
+    justify-content: flex-start;
+    margin-top: 2rem;
+    width: 100%;
+  }
+</style>
+
+<Remote {store} let:data={project}>
+  <div data-cy="remotes-modal" class="container">
+    <Illustration
+      style="margin-bottom: 1.5rem;"
+      variant={IllustrationVariant.Computer} />
+
+    <h1>Manage remotes</h1>
```

Comment:
```suggestion
    <h1>Manage Peers</h1>
```

I think capital P makes sense since it's a header but also happy with "peers"

Reply:
See: https://github.com/radicle-dev/radicle-upstream/pull/1014#issuecomment-717222964.